README.md

```diff
@@ -16,3 +16,4 @@ decentralized and appless.
 ## Getting Started
 
 ## License
+Copyright 2017 - 2018 Lunchclass. and the Absolute project authors. Licensed under Apache License v2.0.
```

Comment:
@faruk13, I think the following sentence is better.

> This project is released under an Apache 2.0 open source license. Please see [LICENSE.md](https://github.com/lunchclass/absolute/blob/master/LICENSE.md).

FYI, you can represent the LICENSE.md link using [MarkDown wiki style](https://guides.github.com/features/mastering-markdown/#examples) as follows.
```[LICENSE.md](https://github.com/lunchclass/absolute/blob/master/LICENSE.md)```

Also, please update your name and your email in [AUTHORS](https://github.com/lunchclass/absolute/blob/master/AUTHORS) file in your first patch.

Thank you for this change again. :)